apps/web/src/components/tx/ConfirmTxDetails/index.tsx

```diff
@@ -106,9 +106,9 @@ export const ConfirmTxDetails = (props: SignOrExecuteProps) => {
       <Divider className={commonCss.nestedDivider} sx={{ pt: 3 }} />
 
       <FormControlLabel
-        sx={{ mt: 2 }}
+        sx={{ mt: 2, mb: -1.3 }}
         control={<Checkbox checked={checked} onChange={handleCheckboxChange} />}
-        label="I understand what I'm signing and acknowledge that this is an irreversible action."
+        label="I understand what I'm signing and know that this is an irreversible action."
```

Comment:
It fits on one line now.

<img width="731" alt="Screenshot 2025-03-21 at 07 24 25" src="https://github.com/user-attachments/assets/d96f79a4-39c5-4a8c-87ff-cb8b28dc8f90" />